lib/trivial.nix

```diff
@@ -108,6 +108,9 @@ rec {
   # Flip the order of the arguments of a binary function.
   flip = f: a: b: f b a;
 
+  # Apply function if argument is non-null
+  mapNullable = f: a: if isNull a then a else f a;
```

Comment:
I can do that, but I've wanted this many times. I bet I could grep for `== null|isNull` and probably find many more uses. I'd be happy to do so after this PR to prove its utility :).